lib/github.js

```diff
@@ -128,7 +128,12 @@ function replaceURL(url, options) {
     });
 }
 
+function setToken(t) {
```

Comment:
There was another PR where I suggested a `require('./lib/github.js)(token)` pattern, but a TODO to follow up would be fine.